doc/faq/troubleshooting_faq.rst

```diff
@@ -149,24 +143,22 @@ tracker only periodically.  If your problem has been determined to be
 a bug and can not be quickly solved, you may be asked to file a bug in
 the tracker so the issue doesn't get lost.
 
-
 .. _git-trouble:
 
 Problems with recent git versions
 =================================
 
-First make sure you have a clean build and install (see
-:ref:`clean-install`), get the latest git update, install it and run a
-simple test script in debug mode::
+First make sure you have a clean build and install (see :ref:`clean-install`),
+get the latest git update, install it and run a simple test script in debug
+mode::
 
-    rm -rf build
     rm -rf /path/to/site-packages/matplotlib*
```

Comment:
This can be replaced by `pip unistall matplotlib`

Reply:
I intentionally left it to cover the case of "seriously broken install" but not sure it's worth it -- let me know what you think.

Reply:
That is a compelling position, including a reference to `pip uninstall matplotlib` someplace would be good though.